crates/context_aware_config/src/api/context/handlers.rs

```diff
@@ -528,15 +554,22 @@ async fn move_handler(
             log::info!("move api failed with error: {:?}", err);
             err
         })?;
-        let version_id = add_config_version(&state, tags, transaction_conn)?;
+        version_id = add_config_version(&state, tags, transaction_conn)?;
         let mut http_resp = HttpResponse::Ok();
 
         http_resp.insert_header((
             AppHeader::XConfigVersion.to_string(),
             version_id.to_string(),
         ));
         Ok(http_resp.json(move_reponse))
```

Comment:
Same here, HttpResponse can be constructed outside the transaction.

Reply:
Same can be done at other places where transactions are being used, can you check and make the changes?

Reply:
Sure let me check